inbox/models/backends/imap.py

```diff
@@ -215,7 +216,21 @@ def categories(self) -> Set[Category]:
         categories.add(self.folder.category)
         return categories
 
-    __table_args__ = (UniqueConstraint("folder_id", "msg_uid", "account_id"),)
+    __table_args__ = (
+        UniqueConstraint("folder_id", "msg_uid", "account_id"),
```

Comment:
This unique constraint with the wrong order of columns is now redundant. I will drop it in a separate PR just to do one thing at a time.